provider/datagen/src/transform/cldr/units/info.rs

```diff
@@ -80,20 +56,17 @@ impl DataProvider<UnitsInfoV1Marker> for crate::DatagenProvider {
                 offset_scientific,
             )?);
 
-            if let Some(units_info_index) = conversion_info_map.get_mut(unit_name.as_str()) {
-                units_info_index.convert_info = Some(convert_unit_index as u16);
-            } else {
-                let units_info_index = UnitsInfoIndex {
-                    dimension: None,
-                    convert_info: Some(convert_unit_index as u16),
-                };
-                conversion_info_map.insert(unit_name.as_str(), units_info_index);
-            }
+            conversion_info_map.insert(unit_name.as_bytes().to_vec(), convert_unit_index);
         }
 
         let result = UnitsInfoV1 {
-            units_info: ZeroMap::from_iter(conversion_info_map),
-            unit_dimensions: VarZeroVec::from(&quantity_vec),
+            units_conversion_map: ZeroTriePerfectHash::try_from(&conversion_info_map)
```

Comment:
Suggestion (optional): I think you might get better results with `ZeroTrieSimpleAscii` than `ZeroTriePerfectHash` because the strings are all lowercase ASCII and there shouldn't be much branching after the first node. This can be changed later though.

Reply:
Okay, I will do it in a followup PR, because this PR is already approved